tests/suites/0_stateless/03_dml/03_0023_insert_into_array.sql

```diff
@@ -0,0 +1,109 @@
+DROP DATABASE IF EXISTS db1;
+CREATE DATABASE db1;
+USE db1;
+
+select '==Array(UInt8)==';
+
+CREATE TABLE IF NOT EXISTS t1(id Int, arr Array(UInt8)) Engine = Memory;
+
+INSERT INTO t1 (id, arr) VALUES(1, [1,2,3]), (2, [254,255]);
+
+select * from t1;
+
+select '==Array(UInt16)==';
+
+CREATE TABLE IF NOT EXISTS t2(id Int, arr Array(UInt16)) Engine = Memory;
+
+INSERT INTO t2 (id, arr) VALUES(1, [1,2,3]), (2, [65534,65535]);
+
+select * from t2;
+
+select '==Array(UInt32)==';
+
+CREATE TABLE IF NOT EXISTS t3(id Int, arr Array(UInt32)) Engine = Memory;
+
+INSERT INTO t3 (id, arr) VALUES(1, [1,2,3]), (2, [4294967294,4294967295]);
+
+select * from t3;
+
+select '==Array(UInt64)==';
+
+CREATE TABLE IF NOT EXISTS t4(id Int, arr Array(UInt64)) Engine = Memory;
+
+INSERT INTO t4 (id, arr) VALUES(1, [1,2,3]), (2, [18446744073709551614,18446744073709551615]);
+
+select * from t4;
+
+select '==Array(Int8)==';
+
+CREATE TABLE IF NOT EXISTS t5(id Int, arr Array(Int8)) Engine = Memory;
+
+INSERT INTO t5 (id, arr) VALUES(1, [1,2,3]), (2, [-128,127]);
+
+select * from t5;
+
+select '==Array(Int16)==';
+
+CREATE TABLE IF NOT EXISTS t6(id Int, arr Array(Int16)) Engine = Memory;
+
+INSERT INTO t6 (id, arr) VALUES(1, [1,2,3]), (2, [-32768,32767]);
+
+select * from t6;
+
+select '==Array(Int32)==';
+
+CREATE TABLE IF NOT EXISTS t7(id Int, arr Array(Int32)) Engine = Memory;
+
+INSERT INTO t7 (id, arr) VALUES(1, [1,2,3]), (2, [-2147483648,2147483647]);
+
+select * from t7;
+
+select '==Array(Int64)==';
+
+CREATE TABLE IF NOT EXISTS t8(id Int, arr Array(Int64)) Engine = Memory;
+
+INSERT INTO t8 (id, arr) VALUES(1, [1,2,3]), (2, [-9223372036854775808,9223372036854775807]);
+
+select * from t8;
+
+select '==Array(Float32)==';
+
+CREATE TABLE IF NOT EXISTS t9(id Int, arr Array(Float32)) Engine = Memory;
+
+INSERT INTO t9 (id, arr) VALUES(1, [1.1,1.2,1.3]), (2, [-1.1,-1.2,-1.3]);
+
+select * from t9;
+
+select '==Array(Float64)==';
+
+CREATE TABLE IF NOT EXISTS t10(id Int, arr Array(Float64)) Engine = Memory;
+
+INSERT INTO t10 (id, arr) VALUES(1, [1.1,1.2,1.3]), (2, [-1.1,-1.2,-1.3]);
+
+select * from t10;
+
+select '==Array(Boolean)==';
+
+CREATE TABLE IF NOT EXISTS t11(id Int, arr Array(Bool)) Engine = Memory;
+
+INSERT INTO t11 (id, arr) VALUES(1, [true, true]), (2, [false, false]), (3, [true, false]), (4, [false, true]);
+
+select * from t11;
+
+select '==Array(String)==';
+
+CREATE TABLE IF NOT EXISTS t12(id Int, arr Array(String)) Engine = Memory;
+
+INSERT INTO t12 (id, arr) VALUES(1, ['aa', 'bb']), (2, ['cc', 'dd']);
+
+select * from t12;
+
+select '==Array(String) Nullable==';
```

Comment:
I think we don't accept `Array(String null)`, so tests should be covered about this type.

Reply:
ok

Reply:
```SQL
CREATE TABLE IF NOT EXISTS t16(id Int, arr Array(String Null)) Engine = Memory; -- {ErrorCode 1005}
```
will parse error, we can't test this case in the stateless test.